t/manifest.t

```diff
@@ -0,0 +1,15 @@
+#!perl -T
+use 5.006;
+use strict;
+use warnings;
+use Test::More;
+
+unless ( $ENV{RELEASE_TESTING} ) {
+    plan( skip_all => "Author tests not required for installation" );
+}
+
+my $min_tcm = 0.9;
+eval "use Test::CheckManifest $min_tcm";
+plan skip_all => "Test::CheckManifest $min_tcm required" if $@;
```

Comment:
Also make sure you have a sufficient version installed on travis. Otherwise this test is useless.

Many people in the company totally rely on travis. Almost nobody runs tests on their devboxes. So, travis tests must include everything.

And most importantly, travis tests should fail if the code generates warnings. Many people just look if the test is green. They don't look at the actual output.